spec/services/merge_users_service_spec.rb

```diff
@@ -248,4 +248,29 @@
       end.to change { prescripteur.reload.user }.from(user2).to(user1)
     end
   end
+
+  context "when target user is responsible for user to merge" do
+    context "when target user has no responsible of her own" do
+      let!(:user_target) { create(:user) }
+      let!(:user_to_merge) { create(:user, responsible: user_target) }
+
+      it "leaves no responsible in the resulting user" do
+        described_class.perform_with(user_target, user_to_merge, [:responsible_id], organisation)
+        expect(user_to_merge).to be_soft_deleted
```

Comment:
si on veut tester la suppression sans tester l'implémentation du soft delete (voire même sans tester la différence entre un soft delete et un hard delete), on pourrait faire ça : 
```suggestion
        expect(User.find_by(id: user_to_merge.id)).to be_nil # Expect user to be deleted
```

Reply:
Ça me va !